about/ob-release-notes.adoc

```diff
@@ -28,4 +28,4 @@ include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
 
-include::modules/ob-release-notes-1-0.adoc[leveloffset=+1]
+include::modules/ob-release-notes-1-1.adoc[leveloffset=+1]
```

Comment:
Looks like it's not building because the `-` here is not matching the `.` in the currently-named `modules/ob-release-notes-1-1.adoc` file.

Guessing it would be better to rename `modules/ob-release-notes-1.1.adoc` to `modules/ob-release-notes-1-1.adoc`, to match how it was in 1.0.